js/crm.menubar.js

```diff
@@ -408,11 +408,17 @@
             return false;
           }
         }
-        // Form redirects to Advanced Search, which does not automatically search with wildcards,
-        // aside from contact name.
-        // To get comparable results, append wildcard to the search term.
-        else if (searchkey !== 'sort_name' && searchkey !== 'id') {
-          $('#crm-qsearch-input').val(searchValue + '%');
+        // Form redirects to Advanced Search, which does not automatically
+        // search with wildcards, aside from contact name and a few other
+        // fields. To get comparable results, append and possible prepend
+        // wildcard to the search term.
+        else if (searchkey !== 'id') {
```

Comment:
@jmcclelland what's the reason for removing the `sort_name` from this condition?

Reply:
I wonder if we should avoid external_id as well?

Reply:
Yes - I agree! I'm about to go camping for a few days but I can work on (and test) that fix in a week.